src/main/java/org/sonarlint/intellij/issue/IssueMatcher.java

```diff
@@ -104,7 +104,7 @@ private static int findEndLineOffset(Document doc, int ijLine, @Nullable Integer
     return lineStart + endOffset;
   }
 
-  private static int findStartLineOffset(PsiFile file, Document doc, int ijLine, @Nullable Integer startOffset) {
+  private static int findStartLineOffset(PsiFile file, Document doc, int ijLine, @Nullable Integer startOffset, int rangeEnd) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 6, down to the maximum allowed 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1142)